CHANGELOG-vitessce-upgrade.md

```diff
@@ -0,0 +1 @@
+- Upgrade Vitessce to 1.1.13.
```

Comment:
It wouldn't hurt to add a note here about what the new version will enable in the Portal.